gravitee-apim-rest-api/gravitee-apim-rest-api-automation/gravitee-apim-rest-api-automation-rest/src/main/resources/open-api.yaml

```diff
@@ -690,6 +703,7 @@ components:
           examples:
             - example-client-id
     ApplicationOAuthClientSettings:
+      type: object
       description: Application OAuth client settings
```

Comment:
maybe a good place to emphasis that this settings require DCR to be enabled / configured at the environment level